bin/create.js

```diff
@@ -263,6 +275,18 @@ function handleCreate (argv) {
                     });
             })
             .then(function (data) {
+                var auth_token = '';
+
+                if (argv.auth) {
+                    try {
+                        auth_token = jwt.sign({}, argv.auth);
```

Comment:
This is just to give the user something to call their webtask with, though it should give the generated secret too/optionally.